client/src/user-context.tsx

```diff
@@ -105,9 +107,14 @@ export function UserDataProvider(props: { children: React.ReactNode }) {
         throw new Error(`${response.status} on ${response.url}`);
       }
       const data = await response.json();
+      const collectionLastModified =
+        data.settings.collections_last_modified_time;
```

Comment:
`data.settings` can be null:

```suggestion
        data.settings?.collections_last_modified_time;
```